src/main/drivers/mco.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+
+#include "platform.h"
+
+#ifdef USE_MCO
+
+#include "drivers/io.h"
+#include "pg/mco.h"
+
+void mcoInit(const mcoConfig_t *mcoConfig)
+{
+    // Only configure MCO2 with PLLI2SCLK as source for now.
+    // Other MCO1 and other sources can easily be added.
+    // For all F4 and F7 varianets, MCO1 is on PA8 and MCO2 is on PC9.
+
+    if (mcoConfig->ioTag[1] == IO_TAG(PC9)) {
```

Comment:
Is there any potential use case where it can work with `mcoConfig->ioTag[1]` not set to PC9? If not then it would make more sense to track this in the config as a bool `mco1_pc9_enabled`, and take the guesswork for users away.

Reply:
I thought this was done deliberately to have it as a resource to block other resources from using pc9. other than that @jflyper has to chime in on this. I pulled in his request to restore the default omnibus 7 target.

Reply:
@mikeller
PC9 is the only pin with MCO2 as AF.

> If not then it would make more sense to track this in the config as a bool mco1_pc9_enabled

We can do that. Let me add the variable. We have to make it build-able for F411 targets and call to `IOInit` anyways.